index.src.html

```diff
@@ -2394,14 +2420,692 @@
+    {{ImageBitmapOptions}}.
+
+: <dfn dict-member for=ImageDecoderInit>desiredWidth</dfn>
+:: Indicates a desired width for decoded outputs. Implementation is best
+    effort; decoding to a desired width may not be supported by all formats/
+    decoders.
+
+: <dfn dict-member for=ImageDecoderInit>desiredHeight</dfn>
+:: Indicates a desired height for decoded outputs. Implementation is best
+    effort; decoding to a desired height may not be supported by all
+    formats/decoders.
+
+ImageDecodeResult Interface {#imagedecoderesult-interface}
+----------------------------------------------------------
+<pre class='idl'>
+<xmp>
+dictionary ImageDecodeResult {
+  required VideoFrame image;
+  required boolean complete;
+};
+</xmp>
+</pre>
+
+: <dfn dict-member for=ImageDecodeResult>image</dfn>
+:: The decoded image.
+
+: <dfn dict-member for=ImageDecodeResult>complete</dfn>
+:: Indicates whether {{ImageDecodeResult/image}} contains the final full-detail
+    output.
+
+    NOTE: {{ImageDecodeResult/complete}} is always `true` when
+        {{ImageDecoder/decode()}} is invoked with |completeFramesOnly| set to
+        `true`.
+
+ImageTrackList Interface {#imagetracklist-interface}
+----------------------------------------------------
+<pre class='idl'>
+<xmp>
+interface ImageTrackList : EventTarget {
+  getter ImageTrack (unsigned long index);
+
+  readonly attribute unsigned long length;
+  readonly attribute long selectedIndex;
+
+  attribute EventHandler onchange;
+};
+</xmp>
+</pre>
+
+### Internal Slots ### {#imagetracklist-internal-slots}
+
+: <dfn attribute for=ImageTrackList>[[track list]]</dfn>
+:: The list of {{ImageTrack}}s describe by this {{ImageTrackList}}.
+
+: <dfn attribute for=ImageTrackList>\[[selected index]]</dfn>
+:: The index of the selected track in {{ImageTrackList/[[track list]]}}. A
+    value of `-1` indeicates that no track is selected.
+
+### Attributes ### {#imagetracklist-attributes}
+
+: <dfn attribute for=ImageTrackList>length</dfn>
+:: The {{ImageTrackList/length}} getter steps are to return the length of
+    {{ImageTrackList/[[track list]]}}.
+
+: <dfn attribute for=ImageTrackList>selectedIndex</dfn>
+:: The {{ImageTrackList/selectedIndex}} getter steps are to return
+    {{ImageTrackList/[[selected index]]}};
+
+: <dfn attribute for=ImageTrackList>onchange</dfn>
+:: An [=event handler IDL attribute=] whose [=event handler event type=] is
+    {{ImageTrackList/change}}.
+
+### Event Summary ### {#imagetracklist-eventsummary}
+
+: <dfn event for=ImageTrackList>change</dfn>
+:: Fired at the {{ImageTrackList}} when the list of {{ImageTrack}}s is
+    established or when attributes of the listed {{ImageTrack}}s are updated.
+
+ImageTrack Interface {#imagetrack-interface}
+--------------------------------------------
+<pre class='idl'>
+<xmp>
+interface ImageTrack {
+  readonly attribute boolean animated;
+  readonly attribute long frameCount;
+  readonly attribute unrestricted float repetitionCount;
```

Comment:
As noted above, maybe we only want to represent discrete counts and leave everything else as zero.